data/category.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/influxdata/influx-stress/write"
 )
 
-func GetDataDirs(root, pattern string) []string {
+func GetDataDirs(root string) []string {
```

Comment:
I don't like the name Data here because that word doesn't really inform the reader of anything specific.

Can we rename Data to Select? I think all the tests under here are ultimately select statements, right?